rskj-core/src/main/java/co/rsk/cli/tools/StartBootstrap.java

```diff
@@ -74,6 +77,16 @@ public Integer call() throws IOException {
         NodeStopper nodeStopper = System::exit;
 
         runBootstrapNode(ctx, preflightChecks, runtime, nodeStopper);
+
+        if (this.stopTimeout > 0L) {
+            try {
+                Thread.sleep(this.stopTimeout);
```

Comment:
could you please explain why we need to "sleep()" here?

Reply:
Just for testing purposes. Right now, when we run the Bootstrap tool, the main thread ends fast and then close the rsk context, making the tool fail when starting the UPD server. So I can't make sure the Bootstrap node really started. So I just added a small timeout to make it wait some time so I can make necessary checks before the node is shut down.